website/common/locales/en/pets.json

```diff
@@ -48,11 +48,10 @@
   "inventoryText": "Click an egg to see usable potions highlighted in green and then click one of the highlighted potions to hatch your pet. If no potions are highlighted, click that egg again to deselect it, and instead click a potion first to have the usable eggs highlighted. You can also sell unwanted drops to Alexander the Merchant.",
   "haveHatchablePet": "You have a <%= potion %> hatching potion and <%= egg %> egg to hatch this pet! <b>Click</b> the paw print to hatch.",
   "quickInventory": "Quick Inventory",
-  "foodText": "food",
-  "food": "Food and Saddles",
-  "noFoodAvailable": "You don't have any Food.",
+  "food": "Pet Food and Saddles",
+  "noFoodAvailable": "You don't have any Pet Food.",
   "noSaddlesAvailable": "You don't have any Saddles.",
-  "noFood": "You don't have any food or saddles.",
+  "noFood": "You don't have any Pet Food or Saddles.",
```

Comment:
I think noFood is no longer used. I can't find it in the code anywhere.
(I went looking through the code to answer a Socialite question and then got curious about this part because I didn't recall any place where you'd see your owned Food and Saddles together. :) )

Reply:
ah right! The Inventory only says "You don't own any of these." Good catch, I'll delete it.